### PR TITLE
About Page - Borja

### PR DESCRIPTION
**Pull request by borjainglefernandez**
## Status
- Done

## GitHub Issues addressed (`#IssueNumber`)
- N/A

## What I did
- Added picture to pfp folder
- Added profile information to team.json

## Screenshots
- Before
image
- After
image

## Tests
- Go to about page and scroll to dev section. Verify that my name and picture are there. Additionally, verify that the Github link works and that no other profiles were impacted.

## Questions/Discussions/Notes
- 

## Merging the PR
- Who should merge this PR?
  - [ ] I will merge it myself
  - [x] The last reviewer to approve can merge it
- Should the commit history be preserved in the base branch, or is it okay to combine all of them into a single commit ("squash-merge")?
  - [ ] preserve the history
  - [x] squash-merge

### CODE REVIEW

File: tcf_website/views/team_info.json [hunk 1]
```diff
@@ -113,6 +113,13 @@
       "class": "2023",
       "img_filename": "ENG_Nikhil_Mahajan.jpg",
       "github": "n4mahajan"
+    },
+    {
+      "name": "Borja Ingle-Fernandez",
+      "role": "Developer",
+      "class": "2023",
+      "img_filename": "ENG_Borja_Ingle-Fernandez.JPG",
```
Comment (Bartleby2718) [inline tcf_website/views/team_info.json:121]: Not terribly important, but for the sake of consistency, this image file seems to be the only one with extension in uppercase.

Comment (borjainglefernandez) [reply tcf_website/views/team_info.json]: Should be fixed now.

**Review by Bartleby2718 — DISMISSED** [review]
Looks good to me! Just a note: based on your commit message (`Add files via upload`), it seems like you used the GitHub web GUI to upload the image file. However, I'd say everything needs to be done (and tested) locally, rather than making changes on the remote (`origin`).

FYI I'm not an exec, so I guess you should wait until an exec approves this PR.

**Comment by borjainglefernandez** [comment]
> Looks good to me! Just a note: based on your commit message (`Add files via upload`), it seems like you used the GitHub web GUI to upload the image file. However, I'd say everything needs to be done (and tested) locally, rather than making changes on the remote (`origin`).
> 
> FYI I'm not an exec, so I guess you should wait until an exec approves this PR.

Right, I just used it this time because I was having issues uploading my picture on PyCharm for some reason. I tested it locally after uploading it to GitHub.

**Review by j-alicia-long — APPROVED** [review]
Looks good to me!

**Pull request merged**